### PR TITLE
feat: CD to Cloudflare Pages

### PR DESCRIPTION
**Pull request by jhult**
Resolves Granola-Team/mina-block-explorer#44

### CODE REVIEW

File: Justfile [hunk 1]
```diff
@@ -60,6 +60,4 @@ dev: build_npm
 
 publish: clean && build_npm
   trunk build --release --filehash true
-  aws cloudfront create-invalidation --distribution-id "$DIST_ID" --paths "/*"
-  aws s3 rm "s3://$BUCKET_NAME" --recursive
-  aws s3 cp dist "s3://$BUCKET_NAME" --recursive
+  npx wrangler pages deploy deploy dist --project-name=mina-search
```
Comment (robinbb) [inline Justfile:63]: Does not match the project name in wrangler.toml. Cause for concern?

**Review by robinbb — CHANGES_REQUESTED** [review]

File: Justfile (hunk 1 above)
Comment (robinbb) [reply Justfile:63]: Wrangler does the cache invalidation?

**Comment by robinbb** [comment]
@jhult Please see my added commits. I ran the updated-by-me version of 'just publish' on my machine. I seem to have been directed to a deployment preview. Does that mean that there's only a preview, or that the deploy took place?

**Comment by robinbb** [comment]
The preview sent me here: https://4278eb7a.minasearch.pages.dev/blocks

**Comment by jhult** [comment]
> @jhult Please see my added commits. I ran the updated-by-me version of 'just publish' on my machine. I seem to have been directed to a deployment preview. Does that mean that there's only a preview, or that the deploy took place?

Those commits look good. For some reason, it did a preview deploy instead of production deployment. Try adding `--branch=production` to the `wrangler...` command.

https://developers.cloudflare.com/workers/wrangler/commands/#deploy-1

https://developers.cloudflare.com/pages/configuration/branch-build-controls/#production-branch-control

**Review by robinbb — APPROVED** [review]

**Pull request merged**